lib/quadquizaminos/contests.ex

```diff
@@ -0,0 +1,107 @@
+defmodule Quadquizaminos.Contests do
+
+
+  @moduledoc """
+  Inserts and gets data from the database that would be used in different functions.
+  -list of all contests
+  -timer
+  -active contests
+  """
+
+  alias Quadquizaminos.Contest
+  alias Quadquizaminos.Repo
+
+  alias Quadquizaminos.Contest.ContestAgent
+  import Ecto.Query, only: [from: 2]
+
+  @doc """
+  Inserts the created contest in the database
```

Comment:
The doc is misleading, the function is responsible for creating a contest not inserting a created contest

Reply:
```suggestion
  creates a contest with the given params
     ## Example
     iex> create_contest(%{name: "ContestB"})
          {:ok,  %Contest{}}
          
      iex> create_contest(%{name: "C"})
            {:error,  changeset}
```